From 2b937aa752b04b3d634fcd3f68745e255626e6a8 Mon Sep 17 00:00:00 2001
From: Adrien Ricciardi <aricciardi@baylibre.com>
Date: Mon, 18 Sep 2023 20:16:56 +0200
Subject: [PATCH 03/38] os: Fixed an unused parameter warning in tpl_null_it().

The 'foo' parameter is not used and triggers a compilation warning.

Signed-off-by: Adrien Ricciardi <aricciardi@baylibre.com>
---
 os/tpl_os_interrupt_kernel.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/os/tpl_os_interrupt_kernel.c b/os/tpl_os_interrupt_kernel.c
index 589bf5e7..10914647 100644
--- a/os/tpl_os_interrupt_kernel.c
+++ b/os/tpl_os_interrupt_kernel.c
@@ -266,6 +266,7 @@ FUNC(tpl_status, OS_CODE) tpl_terminate_isr2_service(void)
 FUNC(void, OS_CODE) tpl_null_it(P2CONST(void, OS_APPL_DATA, AUTOMATIC) foo)
 {
   /* empty function */
+  (void) foo;
 }
 
 #define OS_STOP_SEC_CODE
-- 
2.34.1

